test/lifecycle_test.dart

```diff
@@ -362,6 +393,7 @@ void sharedLifecycleTests<T extends react.Component>({
         expect(
             component.lifecycleCalls,
             equals([
+              matchCall('getChildContext', props: anything, context: anything),
```

Comment:
What are these context-related changes from?

Reply:
We spent some time trying to figure that out. At one point I had to branch off of Keal's wip context branch, and we believe it may be leftovers from that or a reaction to those changes. Ultimately when we looked at the test, the additions made sense and reflect what we'd expect to see!

Reply:
👍